lib/types/BaseAssetContent.ts

```diff
@@ -23,5 +23,10 @@ export interface BaseAssetContent extends KDocumentContent {
 
   metadata?: JSONObject,
 
-  deviceLinks: DeviceLink[]
+  deviceLinks: DeviceLink[],
+
+  category: string,
+  subCategory: string,
+
+
```

Comment:
```suggestion
  subCategory: string,
```

Reply:
Correction seems identical to the original code.

Reply:
I was just removing the extra blank lines, I should have be more precise

Reply:
Done